ogr/ogrsf_frmts/pg/ogrpgdatasource.cpp

```diff
@@ -903,19 +903,7 @@ int OGRPGDataSource::Open(const char *pszNewName, int bUpdate, int bTestOpen,
         }
         OGRPGClearResult(hResult);
 
-        if (sPostGISVersion.nMajor == 0 && sPostGISVersion.nMinor < 8)
-        {
-            // Turning off sequential scans for PostGIS < 0.8
-            hResult = OGRPG_PQexec(hPGConn, "SET ENABLE_SEQSCAN = OFF");
-
-            CPLDebug("PG", "SET ENABLE_SEQSCAN=OFF");
-        }
-        else
-        {
-            // PostGIS >=0.8 is correctly integrated with query planner,
-            // thus PostgreSQL will use indexes whenever appropriate.
-            hResult = OGRPG_PQexec(hPGConn, "SET ENABLE_SEQSCAN = ON");
-        }
+        hResult = OGRPG_PQexec(hPGConn, "SET ENABLE_SEQSCAN = ON");
```

Comment:
Maybe out-of-scope for this PR, but I don't know why GDAL would need to set this at all.

Reply:
yes, I was also wondering. This is obviously a remain of ancient times: https://github.com/OSGeo/gdal/commit/5ffdad199b501ae9e502162d63181f1eacdeaca0 . Removing that now...